uberpoet/filegen.py

```diff
@@ -68,6 +110,81 @@ def gen_file(self, class_count, function_count):
         return FileResult("", [], {})
 
 
+class ObjCHeaderFileGenerator(FileGenerator):
```

Comment:
As mentioned in a comment above, we could maybe combine this with the ObjCSourceFileGenerator below. Open to suggestions.

Reply:
alright keeping it as is!